product-service/__tests__/getProductById.test.ts

```diff
@@ -0,0 +1,49 @@
+import {getProductById} from "../handlers/getProductById";
+import products from "../products";
+
+describe("getProductById", () => {
+    it("should return not found code if product does not exist", async () => {
+        const mockEvent = {
+            pathParameters: {
+                productId: "not-exist"
+            }
+        }
+
+        // @ts-ignore
+        const res = await getProductById(mockEvent)
```

Comment:
Instead of ts-ignore you may use `Partial<APIGatewayProxyEvent>` type for the mockEvent with only the needed properties